src/_scss/layouts/default/header/nav/_mobileHamburger.scss

```diff
@@ -15,16 +15,14 @@
         height: rem(45);
         position: relative;
         width: rem(48);
-
```

Comment:
hover state for hamburger was removed; did design change it so there is no longer a hover state at all?

Reply:
correct! let me find the message

Reply:
it's ok, i believe you
just had to make sure